README.md

```diff
@@ -35,6 +35,15 @@ FileBeat version to use.
 
     filebeat_version: 1.1.1
 
+On Debian based Systems you may use a URL to install a specific .deb
+To do so set 
+
+    filebeat_use_apt_repo: false (default true)
+
+The following deb base url is set by default:
```

Comment:
When `filebeat_use_apt_repo` is set to `false` you can optionally modify the baseurl value to point to another location.